DigitalOffice/Tests/UserServiceTests/CommandsTests/GetUserByIdCommandShould.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using NUnit.Framework;
+using Moq;
+using UserService.Commands;
+using UserService.Commands.Interfaces;
+using UserService.Database.Entities;
+using UserService.Mappers.Interfaces;
+using UserService.Models;
+using UserService.Repositories.Interfaces;
+
+namespace UserServiceUnitTests.CommandsTests
+{
+    public class GetUserByIdCommandShould
+    {
+        private IGetUserByIdCommand getUserInfoByIdCommand;
+        private Mock<IGetUserByIdRepository> repositoryMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<IGetUserByIdRepository>();
+            getUserInfoByIdCommand = new GetUserByIdCommand(repositoryMock.Object);
+        }
+
+        [Test]
+        public void Throw_Exception_If_Repository_Returns_Null()
```

Comment:
Rename, underscores not acceptable in method name

Reply:
fixed